### PR TITLE
Add bidfloor param

### PR DESCRIPTION
**Pull request by Kamoris**

### CODE REVIEW

File: modules/rtbhouseBidAdapter.js [hunk 1]
```diff
@@ -96,7 +96,8 @@ function mapImpression(slot) {
     id: slot.bidId,
     banner: mapBanner(slot),
     native: mapNative(slot),
-    tagid: slot.adUnitCode.toString()
+    tagid: slot.adUnitCode.toString(),
+    bidfloor: slot.params.bidfloor
```
Comment (piwanczak) [inline modules/rtbhouseBidAdapter.js]: tryparse int/float maybe? There might be something riddiculously unexpected, needs sanitization imho

File: test/spec/modules/rtbhouseBidAdapter_spec.js [hunk 2]
```diff
@@ -132,6 +132,14 @@ describe('RTBHouseAdapter', () => {
       expect(data.source.tid).to.equal('example-transaction-id');
     });
 
+    it('should include bidfloor in request', () => {
```
Comment (piwanczak) [inline test/spec/modules/rtbhouseBidAdapter_spec.js]: description "... if available" 
We are not making it required, no?

**Review by piwanczak — CHANGES_REQUESTED** [review]

File: modules/rtbhouseBidAdapter.js (hunk 1 above)
Comment (Kamoris) [reply modules/rtbhouseBidAdapter.js]: Good point, fixed.

File: test/spec/modules/rtbhouseBidAdapter_spec.js (hunk 2 above)
Comment (Kamoris) [reply test/spec/modules/rtbhouseBidAdapter_spec.js]: Added.

**Pull request merged**